lapce-app/src/editor.rs

```diff
@@ -290,6 +285,7 @@ impl EditorData {
             }
         }
 
+        println!("run edit command");
```

Comment:
Are all the print lines intentionally left in 😃 ?

Reply:
left by accident :)